orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -85,9 +90,16 @@ class DependentPipelineStarter implements ApplicationContextAware {
 
     def pipeline
 
+    Map mdc = null
     def t1 = new Thread(new Runnable() {
       @Override
       public void run() {
+        log.debug("MDC destination thread 1: " + MDC.getCopyOfContextMap())
```

Comment:
Manually copying around the MDC context seems a little suspect, would the following work?

```
    def runnable = AuthenticatedRequest.propagate({
      if (parentPipeline.executionEngine == Execution.V2_EXECUTION_ENGINE) {
        pipeline = pipelineLauncher().start(json)
      } else {
        pipeline = pipelineStarter.start(json)
      }
    }, true, principal) as Runnable

    def t1 = new Thread(runnable)
    t1.start()
```